### DISCUSSION TITLE
Arrow key mapping

### DISCUSSION THREAD

**Discussion by fantomac** [CommandPost/CommandPost#3253] [Technical Support]
Hi,
Probably a stupid question, but I can't mapp the CONTROL and RIGHT command because right arrows isn't in the list.
I just want to assign virtual desktop right or left to my Midi input key. 
Thanks in advance.
And thanks for your work 👍 !

**Comment by latenitefilms** [comment]
Did you try the **CONTROL and Right** action?

<img width="1280" alt="image" src="https://github.com/CommandPost/CommandPost/assets/22286696/d2554119-bb73-42c5-aa5e-b7da1749df4d">

**Reply by fantomac** [reply]
Thanks,
Yes I tried but doesn't work.
But... with the second one (Triggers a virtual keyboard press using an alternative method) it's working 👍

Another question.
I want to use the slider or the knob to increase or decrease a value, how can I configure it ?
Sorry to ask but the help section is not available... (http://help.commandpost.io)

Thanks in advance 🙏
